edit/buildozer_test.go

```diff
@@ -205,6 +210,47 @@ func TestTargetExpressionToBuildFiles(t *testing.T) {
 	runTestTargetExpressionToBuildFiles(t, "BUILD.bazel")
 }
 
+func runTestAppendCommands(t *testing.T, buildFileName string) {
+	tmp := setupTestTmpWorkspace(t, buildFileName)
+	defer os.RemoveAll(tmp)
+
+	for _, tc := range []targetExpressionToBuildFilesTestCase{
+		{tmp, ".:__pkg__", []string{"./" + buildFileName}},
+		{tmp, "a" + ":__pkg__", []string{"a/" + buildFileName}},
+		{"", "a" + ":__pkg__", []string{"a/" + buildFileName}},
+	} {
+		if tc.rootDir == "" {
+			cwd, err := os.Getwd()
+			if err != nil {
+				t.Fatal(err)
+			}
+			// buildozer should be able to find the WORKSPACE file in the current wd
+			if err := os.Chdir(tmp); err != nil {
+				t.Fatal(err)
+			}
+			defer os.Chdir(cwd)
+		}
+
+		commandsByFile := make(map[string][]commandsForTarget)
+		opts := NewOpts()
+		opts.RootDir = tc.rootDir
+		appendCommands(opts, commandsByFile, tc.buildFiles)
+		if len(commandsByFile) != 1 {
+			t.Errorf("Expect one target after appendCommands")
+		}
+		for _, value := range commandsByFile {
+			if value[0].target != tc.target {
+				t.Errorf("appendCommands for buildfile %s yielded target %s, expected %s", tc.buildFiles, value[0].target, tc.target)
+			}
+		}
+	}
+}
+
+func TestAppendCommands(t *testing.T) {
+        runTestAppendCommands(t, "BUILD")
+        runTestAppendCommands(t, "BUILD.bazel")
```

Comment:
Please iterate over `BuildFileNames` instead, like here: https://github.com/bazelbuild/buildtools/pull/838/files#diff-10c338c9802b1617e469141df410c9ebL204

Reply:
Sure! I copy-pasted the test right above (TestTargetExpressionToBuildFiles line 203, do you want me to update in the same PR?)

Reply:
Nevermind I see that the other PR is actually updating the code I copy/pasted!